packages/components/src/overlays/Popover/docs/Popover.stories.tsx

```diff
@@ -0,0 +1,189 @@
+            <p>Popover content</p>
+            <ButtonGroup>
+                <Button variant="secondary">Next</Button>
+                <Button>Go it</Button>
+            </ButtonGroup>
+        </>
+    }
+};
+
+/**
+ * A popover can have footer text.
+ */
+export const FooterText: Story = {
+    args: {
+        children: <>
+            <Text>Title</Text>
+            <p>Popover content</p>
+            <Footer>
+                All right reserved.
+            </Footer>
+            <Button>Go it</Button>
+        </>
+    }
+};
+
+/**
+ * A popover can open at different positions..
+ */
+export const Position: Story = {
+    args: {
+        placement: "right",
+        children: <>
+            <Text>Title</Text>
+            <p>Popover content</p>
+        </>
+    }
+};
+
+/**
+ * A popover will not open when its trigger is disabled.
+ */
+export const DisabledTrigger: Story = {
+    render: () =>
+        <PopoverTrigger>
+            <Button isDisabled variant="secondary" aria-label="information"><InfoIcon /></Button>
+            <Popover>
+                <Text>Title</Text>
+                <p>Popover content</p>
+            </Popover>
+        </PopoverTrigger>
+};
+
+const HighlightedTrigger = ({ children }: { children: ReactNode }) => {
+    const triggerRef = useRef(null);
+    const [isOpen, setOpen] = useState(false);
+
+
+    return (
+        <SlotProvider values={[
+            [PopoverContext, { isOpen, onOpenChange: setOpen }],
+            [ButtonContext, {
+                onPress: () => setOpen(!isOpen),
+                ref: triggerRef,
+                variant: isOpen ? "primary" : "secondary"
+            }]
+        ]}
+        >
+            {children}
+        </SlotProvider>
+    );
+};
+/**
+ * A popover isOpen state or close function can be retrieved from PopoverContext and ButtonContext.
+ **/
+export const Context: Story = {
+    render: () =>
+        <HighlightedTrigger>
+            <PopoverTrigger>
+                <Button aria-label="information"><InfoIcon /></Button>
+                <Popover>
+                    <Text>Title</Text>
+                    <p>Popover content</p>
+                </Popover>
+            </PopoverTrigger>
+        </HighlightedTrigger>
+};
+
+const ControlledPopover = () => {
+    const [isOpen, setOpen] = useState(false);
+    const triggerRef = useRef(null);
+
+    return (
+        <PopoverTrigger>
+            <Inline>
+                <Button onPress={() => setOpen(true)} variant="secondary" aria-label="information"><InfoIcon /></Button>
+                <span ref={triggerRef}>
+ Popover will be positioned relative to me
+                </span>
+            </Inline>
+            <Popover triggerRef={triggerRef} isOpen={isOpen} onOpenChange={setOpen}>
+                <Text>Title</Text>
+                <p>Popover content</p>
+            </Popover>
+        </PopoverTrigger>
+    );
+};
+
+/**
+ * A popover open state can be handled in controlled mode.
```

Comment:
```suggestion
 * A popover's open state can be handled in controlled mode.
```
also, you used isOpen in the previous example. It should probably be the same in both examples.